keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -166,15 +166,22 @@ func RunningModule() (types.ModuleRunning, error) {
 
 // GetLatestVersion return the latest non-prerelease, non-draft version of kubeedge in releases
 func GetLatestVersion() (string, error) {
-	//Download the tar from repo
-	versionURL := "curl -k " + latestReleaseVersionURL
-	cmd := exec.Command("sh", "-c", versionURL)
-	latestReleaseData, err := cmd.Output()
+	// curl https://kubeedge.io/latestversion
+	resp, err := http.Get(latestReleaseVersionURL)
 	if err != nil {
-		return "", err
+		return "", fmt.Errorf("failed to get latest version from %s: %v", latestReleaseVersionURL, err)
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusOK {
+		return "", fmt.Errorf("failed to get latest version from %s, expected %d, got status code: %d", latestReleaseVersionURL, http.StatusOK, resp.StatusCode)
 	}
 
-	return string(latestReleaseData), nil
+	body, err := ioutil.ReadAll(resp.Body)
```

Comment:
Could use `io.ReadAll` instead of `ioutil.ReadAll`?
After golang 1.16, it is more recommended to use `io` instead of `ioutil`

Reply:
Done! I found another ioutil call in the same file as well to fix as well